### PR TITLE
add v0.6.1 release notes

### PR DESCRIPTION
**Pull request by hao1939**

### CODE REVIEW

File: release-note.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Improvements:
+    Faster loading of the job list UI: Now the page gets 5x faster than before when loading its content.
+
+Bug fixes: (Omitted here)
+
+Known issues:
+    #827 Deploy PAI master and worker on the same node may lead to resource competition.
+    #813 Still in investigation. Install PAI on some old kernel may fail.
+    #713 Yarn can't use all the resources (for example GPUs) listed on PAI dashboard, it's some kind of misleading.
```
Comment (fanyangCS) [inline release-note.txt]: >Yarn can't use all the resources (for example GPUs) listed on PAI dashboard, it's some kind of misleading. [](start = 9, length = 106)

Yarn may not use all the resource shown on the PAI dashboard, due to configuration issues. This is an expected behavior.

File: release-note.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Release v0.6.1 release notes.
+
+New features:
+    The 'paictl' tool: Introducing paictl, the universal deployment/management tool with the functionalities of image building, service start/stop, k8s bootup/clean, and configuration generation.
+    Single-box deployment: Support deploying on a single machine.
```
Comment (fanyangCS) [inline release-note.txt]: >Support deploying on a single machine [](start = 27, length = 37)

support single-box deployment for evaluation purpose.

File: release-note.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Release v0.6.1 release notes.
+
+New features:
+    The 'paictl' tool: Introducing paictl, the universal deployment/management tool with the functionalities of image building, service start/stop, k8s bootup/clean, and configuration generation.
```
Comment (fanyangCS) [inline release-note.txt]: >universal [](start = 47, length = 9)

remove

File: release-note.txt [hunk 4]
```diff
@@ -0,0 +1,17 @@
+Release v0.6.1 release notes.
```
Comment (hwuu) [inline release-note.txt]: OpenPAI v0.6.1 release notes.

BTW, isn’t it 0.6.0?

File: release-note.txt (hunk 1 above)
Comment (hao1939) [reply release-note.txt]: Done

File: release-note.txt (hunk 2 above)
Comment (hao1939) [reply release-note.txt]: Done

File: release-note.txt (hunk 3 above)
Comment (hao1939) [reply release-note.txt]: Done

File: release-note.txt (hunk 4 above)
Comment (hao1939) [reply release-note.txt]: yeah, we already used the version number 0.6.0 (although not announced)

Comment (hao1939) [reply release-note.txt]: I renamed to release-notes.txt.
But not changed to .md file. 

Maybe raw txt is enough for us?

File: release-notes.txt [hunk 5]
```diff
@@ -0,0 +1,17 @@
+Release v0.6.1 release notes.
+
+New features:
+    The 'paictl' tool: Introducing paictl, the deployment/management tool with the functionalities of image building, service start/stop, k8s bootup/clean, and configuration generation.
+    Single-box deployment: Support single-box deployment for evaluation purpose..
```
Comment (fanyangCS) [inline release-notes.txt]: >. [](start = 80, length = 1)

duplicated. please remove

**Pull request merged**